src/main/perl/CLI.pm

```diff
@@ -6,7 +6,7 @@ use CAF::Reporter 16.8.1 qw($LOGFILE);
 use CAF::Lock qw(FORCE_IF_STALE FORCE_ALWAYS);
 use CAF::Object qw (SUCCESS throw_error);
 use EDG::WP4::CCM::CacheManager;
-use EDG::WP4::CCM::Fetch qw(NOQUATTOR NOQUATTOR_EXITCODE NOQUATTOR_FORCE);
+use EDG::WP4::CCM::Fetch::Config 16.8.1 qw(NOQUATTOR NOQUATTOR_EXITCODE NOQUATTOR_FORCE);
```

Comment:
it's more recent than 16.8.0 release and when you make rpms from the current master, you get 16.8.1-SNAPSHOT....; so this allows you to ask for something more recent than 16.8 and still be able to test it.